src/which.zig

```diff
@@ -133,7 +140,20 @@ fn whichWin(buf: *bun.WPathBuffer, path: []const u8, bin: []const u8) ?[:0]const
         return searchBin(buf, bin_utf16.len, check_windows_extensions);
     }
 
-    if (path.len == 0) return null;
+    // check if bin is in cwd
+    if (bun.strings.containsChar(bin, '/')) {
+        if (searchBinInPath(
+            buf,
+            &path_buf,
+            cwd,
+            bun.strings.withoutPrefixComptime(bin, "./"),
```

Comment:
same here

Reply:
this was from reverting https://github.com/oven-sh/bun/pull/9691. i'll look if this is correct code.